tests/dist.rs

```diff
@@ -496,24 +498,50 @@ fn uninstall(
     Ok(())
 }
 
+#[derive(Copy, Clone, Debug)]
+enum Compressions {
```

Comment:
just listing the compressions in an enum in vec![] syntax wouldn't be much longer and would allow more combinations in future. This is fine, just food for thought.

Reply:
Yeah, I think ultimately we'll want either a vec, or some kind of bitfield or somesuch, for now as you say, this will do.